docs/explanations/animation.md

```diff
@@ -28,15 +28,12 @@ nframes = 30
 framerate = 30
 hue_iterator = range(0, 360, length=nframes)
 
-record(fig, "color_animation.mp4", hue_iterator;
+Record(fig, hue_iterator;
```

Comment:
In this case it seems like Franklin isn't displaying the output - that should probably be corrected...

Reply:
https://github.com/MakieOrg/Makie.jl/blob/f51eb527806e184690b577e17e630e9c9c521cb9/docs/utils.jl#L93-L99 

needs some kind of dispatch on `VideoStream` objects, to return gifs...